Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.yml

```diff
@@ -3,797 +3,957 @@ commonfields:
+          description: The name of the domain administrator.
+          type: String
+        - contextPath: Domain.Admin.Phone
+          description: The phone number of the domain administrator.
+          type: String
+        - contextPath: Domain.Registrant.Country
+          description: The country of the registrant.
+          type: String
+        - contextPath: Domain.Registrant.Email
+          description: The email address of the registrant.
+          type: String
+        - contextPath: Domain.Registrant.Name
+          description: The name of the registrant.
+          type: String
+        - contextPath: Domain.Registrant.Phone
+          description: The phone number for receiving abuse reports.
+          type: String
+        - contextPath: Domain.Tags
+          description: (List) Tags of the domain.
+          type: Unknown
+        - contextPath: Domain.WHOIS.DomainStatus
+          description: The status of the domain.
+          type: String
+        - contextPath: Domain.WHOIS.NameServers
+          description: (List<String>) Name servers of the domain.
+          type: String
+        - contextPath: Domain.WHOIS.CreationDate
+          description: The date that the domain was created.
+          type: Date
+        - contextPath: Domain.WHOIS.UpdatedDate
+          description: The date that the domain was last updated.
+          type: Date
+        - contextPath: Domain.WHOIS.ExpirationDate
+          description: The expiration date of the domain.
+          type: Date
+        - contextPath: Domain.WHOIS.Registrant.Name
+          description: The name of the registrant.
+          type: String
+        - contextPath: Domain.WHOIS.Registrant.Email
+          description: The email address of the registrant.
+          type: String
+        - contextPath: Domain.WHOIS.Registrant.Phone
+          description: The phone number of the registrant.
+          type: String
+        - contextPath: Domain.WHOIS.Registrar.Name
+          description: 'The name of the registrar, for example: `GoDaddy`'
+          type: String
+        - contextPath: Domain.WHOIS.Registrar.Email
+          description: The email address of the contact.
+          type: String
+        - contextPath: Domain.WHOIS.Registrar.Phone
+          description: The phone number of contact.
+          type: String
+        - contextPath: Domain.WHOIS.Admin.Name
+          description: The name of the domain administrator.
+          type: String
+        - contextPath: Domain.WHOIS.Admin.Email
+          description: The email address of the domain administrator.
+          type: String
+        - contextPath: Domain.WHOIS.Admin.Phone
+          description: The phone number of the domain administrator.
+          type: String
+    - arguments:
+        - name: new_argument
+          description: description
+          required: false
+          isArray: false
+          defaultValue: ""
+      name: securitytrails-sql-query
```

Comment:
Not sure where are the arguments of the new function 'securitytrails-sql-query'. 
I expected to see ‘sql’ and ‘timeout’ args.